### ISSUE TITLE
outputting `esc.beep` to ansi-diff-stream  will cause the output of the current line to be cleared

### ISSUE THREAD

**Issue by jacobbubu** [derhuerst/prompt-skeleton#1]
On the lines 48 and 57, outputting `esc.beep` to ansi-diff-stream  will cause the output of the current line to be cleared.

You may use `process.stdout.write(esc.beep)` to instead of.

**Comment by derhuerst** [comment]
Isn't this a bug in `ansi-diff-stream`?

**Comment by jacobbubu** [comment]
I don't think so. `ansi-diff-stream` calculates the difference according to the contents of two successive calls. When you send `esc.beep` to `p.out.write`, it will completely overwrite the content of the previous call.

**Comment by derhuerst** [comment]
> When you send `esc.beep` to `p.out.write`, it will completely overwrite the content of the previous call.

I don't see why it should do that.

**Comment by jacobbubu** [comment]
Because when calculating the difference between two frames of data, if the data contains ansi codes, it will be difficult. But of course, it can do a special processing for beeping.

**Comment by derhuerst** [comment]
I'd say that, because the package is called `ansi-diff-stream`, it ought to diff ANSI, including edge cases like `esc.beep`.
